src/languages/smarty.coffee

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+
+  name: "Smarty"
+  description: "Smarty"
+  namespace: "smarty"
+  fallback: ['html', 'js', 'php']
+
+  ###
+  Supported Grammars
+  ###
+  grammars: [
+    "HTML (Smarty)"
+  ]
+
+  ###
+  Supported extensions
+  ###
+  extensions: [
+    'smarty'
+  ]
+
+  options: []
+
```

Comment:
Do you know which of the existing beautifiers support Smarty?

If so, you should add, for example

```javascript
defaultBeautifier: "Pretty Diff"
```

in which case `Pretty Diff` should be the beautifiers name that supports Smartly.